Networking/NetworkingTests/Mapper/OrderListMapperTests.swift

```diff
@@ -156,6 +185,12 @@ private extension OrderListMapperTests {
         return mapOrders(from: "orders-load-all")
     }
 
+    /// Returns the OrderListMapper output upon receiving `orders-load-all-without-data`
```

Comment:
super nit:
```suggestion
    /// Returns the [Order] output upon receiving `orders-load-all-without-data`
```

Reply:
Updated in 7848519638b4e92116278ea02395cd4122d90974.